### PR TITLE
Fixes for common errors

### PR DESCRIPTION
**Pull request by uF4No**
Matches code with latest fixes in tutorial ([PR in docs](https://github.com/matter-labs/zksync-web-v2-docs/pull/110)), specifically:

- increase amount to fund paymaster to 0.03ETH
- adds comment to manually include gasLimit in `use-paymaster`
- updates dependencies to latest versions

Also added a basic README with links to the docs/tutorial

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## Installation and compilation
+
+You need Node.js and Yarn.
+
+Install all dependencies with `yarn`.
+
+Compile contracts with `yarn hardhat compile`
+
+## Deploymenta and usage
```
Comment (krofax) [inline README.md:15]: Do you mean, Deployment and usage?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Code for the "Build a custom paymaster" tutorial from the [zkSync v2 documentation](https://v2-docs.zksync.io/dev/).
+
+You can find a full step-by-step guide to build this project [in this article](https://v2-docs.zksync.io/dev/tutorials/custom-paymaster-tutorial.html#prerequisite).
+
+## Installation and compilation
+
+You need Node.js and Yarn.
+
+Install all dependencies with `yarn`.
```
Comment (krofax) [inline README.md:11]: I'd rather say "Install all dependencies with `yarn add` "

File: README.md (hunk 1 above)
Comment (uF4No) [reply README.md]: fixed in latest commit

File: README.md (hunk 2 above)
Comment (uF4No) [reply README.md]: fixed in latest commit

**Pull request merged**